src/server/routes/auth-routes.js

```diff
@@ -1,78 +1,112 @@
 import express from 'express';
+import models from '../models';
+import crypto from 'crypto';
+import session from 'express-session';
+import jwt from 'jsonwebtoken';
+import config from '../config/testing';
+
 const routes = express.Router();
+const secret = config.secret; //TODO: Store secret elsewhere and change it!!
+const accessToken = config.accessToken;
 
-import models from "../models";
+// **************************** Login *******************************
 
-var passport = require('passport');
-var LocalStrategy = require('passport-local').Strategy;
-var session = require('express-session');
+var login = (user, res) => {
+  var token = jwt.sign(user, secret, {
+    expiresInMinutes: 1440 // expires in 24 hours
+  });
 
-routes.use(session({ secret: 'superSeCret' }));
-routes.use(passport.initialize());
-routes.use(passport.session());
+  user.updateAttributes({
+    last_login: new Date()
+  });
 
-// **************************** Login *******************************
+  res.cookie(accessToken, token, { maxAge: 86400000, httpOnly: true }); //might need more security to defend against CSRF:
+  //see : https://stormpath.com/blog/where-to-store-your-jwts-cookies-vs-html5-web-storage/
 
-passport.serializeUser(function(user, done) {
-  done(null, user.id);
-});
+  return res.json({
+    success: true
+  });
+};
+
+var currentUser = req => {
```

Comment:
It might be useful to add these as properties on the request object. We could then use it in other routes outside this file (unless you want to export these functions but I think havig them on the request object is cleaner).